core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineYugabyte.java

```diff
@@ -0,0 +1,10 @@
+package com.scalar.db.storage.jdbc;
+
+import java.sql.Driver;
+
+class RdbEngineYugabyte extends RdbEnginePostgresql {
+  @Override
+  public Driver getDriver() {
+    return new com.yugabyte.Driver();
```

Comment:
It looks like we don't use HikariCP for YugabyteDB. So, HikariCP is not required for YugabyteDB?

Reply:
Right.

The combination of Apache Commons DBCP and YugabyteDB driver isn't introduced as an example on the official document while HikariCP is introduced. But YugabyteDB support team said we can use DBCP as well. I also looked over the source code of the YugabyteDB driver and it seems to work well.

As for HikariCP, I think we can integrate it with the following steps as a separate project:
1. Benchmark ScalarDB with all the JDBC storages using HikariCP
2. If the performance of 1 isn't significantly better than the current performance, we don't need to move forward on it
3. Introduce a new property `scalar.db.jdbc.connection_pool.library` or something which can be either of `dbcp` or `hikaricp`
4. Use `dbcp` by default in version 3 and use `hikari` by default in version 4

Reply:
Sounds good! Thanks.